packages/block-library/src/list/editor.scss

```diff
@@ -3,3 +3,83 @@
 	padding-left: 1.3em;
 	margin-left: 1.3em;
 }
+
+.editor-styles-wrapper .block-library-list ol[type="1"] {
```

Comment:
In SCSS you can group similar rules together. 

.class-name {
    tag1 { ... }
    tag2 { ... }
}

Is this editor-style-wrapper class necessary here? I guess it might be, I wanted to double check.

Reply:
The CSS styles have been updated :)